hepstats/hypotests/calculators/asymptotic_calculator.py

```diff
@@ -224,7 +224,7 @@ def pnull(self, qobs, qalt=None, onesided=True, onesideddiscovery=False, qtilde=
         return pnull
 
     def qalt(self, poinull, poialt, onesided, onesideddiscovery) -> np.array:
-        """ Compute alternative values of the $$\\Delta$$ log-likelihood test statistic using the asimov
+        """ Compute alternative hypotesis values of the $$\\Delta$$ log-likelihood test statistic using the asimov
```

Comment:
hypothesis

Reply:
Note that the typo is several times in this commit, in other files.

Reply:
Ok indeed I made several copy/pastes of the same typo ... thanks for spotting